server/cpho/views/indicators.py

```diff
@@ -43,12 +43,34 @@ class IndicatorForm(ModelForm):
     class Meta:
         model = Indicator
         fields = "__all__"
-        widgets = {
-            "relevant_period_types": forms.CheckboxSelectMultiple,
-        }
+        # widgets = {
+        #     "relevant_period_types": forms.CheckboxSelectMultiple,
+        # }
+
+    def __init__(self, *args, **kwargs):
+        user = None
+        if "user" in kwargs:
+            user = kwargs.pop("user", None)
+        super(IndicatorForm, self).__init__(*args, **kwargs)
```

Comment:
super with arguments is an outdated python2 thing and an anti-pattern in python3 (unless you want to wacky things like skip the parent class to call the grandparent). Just use `super().__init__(*args,**kwargs)`